demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -192,3 +211,18 @@ def to_id_set_entity(self) -> dict:
         id_set_entity["file_path"] = str(self.path)
         id_set_entity["pack"] = self.in_pack.object_id  # type: ignore[union-attr]
         return id_set_entity
+
+    def is_incident_to_alert(self, marketplace: MarketplaceVersions) -> bool:
+        """
+        As long as the content item does not have an implementation of the `is_incident_to_alert` function,
```

Comment:
can we replace this with a `raise NotImplementedError`? Not too important, but we can remove this attribute this way. Let's keep it to the end of all other comments though.